samples/hdfs/fluid/app.yaml

```diff
@@ -8,5 +8,8 @@ spec:
       restartPolicy: OnFailure
       containers:
         - name: fluid-hdfs-demo
-          image: registry.cn-beijing.aliyuncs.com/yukong/fluid-hdfs-demo:1.0.0
+          image: registry.cn-hangzhou.aliyuncs.com/qiulingwei/fluid-hdfs-demo:1.2.0
           imagePullPolicy: Always
+          env:
+          - name: HADOOP_PORT
+            value: "20000"
```

Comment:
change "20000" -> "19998"?

Reply:
I write the port number at will, because the users need to fill in according to the actual port number found